### DISCUSSION TITLE
Support muting posts from a person

### DISCUSSION THREAD

**Discussion by digeomel** [dailydotdev/daily#1150] [Feature Requests]
Please support muting posts from a person. I don't want to get any notifications about their posts. I don't care if they are group/page admins that I am member of.
Thank you.

**Comment by nimrodkra** [comment]
I case you're referring to the option to block a specific person, that's currently not supported. We may decide to build it down the road. However, in case you're bothered by receiving notifications about new posts in Squads that you've joined, you can easily disable those by clicking the Squad notifications settings (top right corner on the Squad's homepage):

![Screenshot 2024-05-06 at 16 11 49](https://github.com/dailydotdev/daily/assets/41470823/981d4b71-88c4-406b-9b20-f6e0f261875b)

And then disabling notifications for new posts: 

![Screenshot 2024-05-06 at 16 12 54](https://github.com/dailydotdev/daily/assets/41470823/d0df5fd6-27f3-415e-b635-bad1814bc169)

Hope this helps!
